saged/keep_ratios.py

```diff
@@ -208,19 +227,20 @@ def subset_to_equal_ratio(train_data: datasets.LabeledDataset,
             val_data.reset_filters()
 
     with open(args.out_file, 'w') as out_file:
-        out_file.write('accuracy\ttrain studies\ttrain samples\tval samples\ttrain sample count\t')
-        out_file.write('fraction of data used\n')
-        for (accuracy,
-             train_study_str,
-             train_sample_str,
-             val_sample_str,
-             supervised_train_samples,
-             percent) in zip(accuracies,
-                             supervised_train_studies,
-                             supervised_train_sample_names,
-                             supervised_val_sample_names,
-                             supervised_train_sample_counts,
-                             subset_percents,
-                             ):
-            out_file.write(f'{accuracy}\t{train_study_str}\t{train_sample_str}\t')
-            out_file.write(f'{val_sample_str}\t{supervised_train_samples}\t{percent}\n')
+        # Write header
+        out_file.write('accuracy\tbalanced_accuracy\tf1_score\ttrain studies\ttrain samples\t')
+        out_file.write('val samples\ttrain sample count\tfraction of data used\n')
```

Comment:
I think you should just be able to do this here:
```suggestion
        out_file.write('accuracy\tbalanced_accuracy\tf1_score\ttrain studies\ttrain samples\t'
                       'val samples\ttrain sample count\tfraction of data used\n')
```
(the strings should automatically be concatenated).